stories/Who_was_protected_by_the_first_COVID-19_lockdown.ipynb

```diff
@@ -0,0 +1,2412 @@
+    "\n",
+    "Each of our stories comes in the form of a Jupyter notebook, which will contain all the code required to follow along with the analysis, along with an explanations of our thought process."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Covid19 and Deprivation\n",
+    "\n",
+    "Everyone in the UK has been impacted by the COVID-19 pandemic, from London to Manchester, Cardiff to Edinburgh. The virus has affected everyone, and taken many of the lives of people across Country.\n",
+    "\n",
+    "On the [23rd of March 2020](https://www.bbc.co.uk/news/uk-52012432), the UK Government announced various lockdown measures with the intention of limiting the spread of the virus and reducing the number of COVID-19 related deaths. These lockdown measures meant the temporary closure of many commercial shops and businesses, as well as the limiting of work based travel to only those jobs that could not be done at home.\n",
+    "\n",
+    "**We are concerned that the impact of COVID-19 has disproportionately affected certain groups of people. In particular, that the lockdown measure may have had a worse impact for those in the most deprived areas, whose livelihoods may have required them to leave the house more frequently.**\n",
+    "\n",
+    "There have been a number of [concerns](https://www.ft.com/content/301c847c-a317-4950-a75b-8e66933d423a) with Government COVID-19 reporting, in particular with testing and mortality statistics. This motivates independent, open analysis to validate and expand on our understanding of our current state of the pandemic.\n",
+    "\n",
+    "Earlier in June, the Office of National Statistics (ONS) [published a report](https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/bulletins/deathsinvolvingcovid19bylocalareasanddeprivation/latest) exploring this exact question: to assess whether those living in the most deprived areas of the UK were disproportionately affected by COVID-19. The report seems to confirm our fear - between the months of March to May 2020 those in the most deprived areas of the UK were more than twice as likely to die as a result of COVID-19 than those in the least deprived areas.\n",
+    "\n",
+    "There are two caveats that we have with the ONS analysis. The first is **reproducibility**. We want to confirm the ONS results by making analysis procedure open. The second caveat is that the ONS report **aggregates data over time**, and therefore that it might miss interesting differences in outcomes between the different stages of lockdown. Between March and May represents the time when the lockdown was most severe, with measures relaxing from June onwards. We wonder whether the ONS analysis will continue to be relevant as lockdown eases. For this purpose, we wish to extend the ONS analysis to cover all available data, and at the same time, make a comparison between the different stages of lockdown.\n",
```

Comment:
```suggestion
    "There are two caveats that we have with the ONS analysis. The first is **reproducibility**. We want to confirm the ONS results by making the analysis procedure open. The second caveat is that the ONS report **aggregates data over time**, and therefore that it might miss interesting differences in outcomes between the different stages of lockdown. The lockdown was most severe between March and May, with measures relaxing from June onwards. We wonder whether the ONS analysis will continue to be relevant as lockdown eases. For this purpose, we wish to extend the ONS analysis to cover all available data, and at the same time, make a comparison between the different stages of lockdown.\n",
```